samples/graph-meeting-notification/csharp/MeetingNotification/Helper/SubscriptionManager.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace MeetingNotification.Helper
+{
+    using MeetingNotification.Model.Configuration;
+    using MeetingNotification.Provider;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+    using Microsoft.Graph;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http.Headers;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class SubscriptionManager : BackgroundService
```

Comment:
Looks like this is a background service which will be running on a particular time.
Name is somehow confusing :) but it's fine.